.gitignore

```diff
@@ -97,9 +98,11 @@ total.coverage/
 coverage_percent.txt
 
 #build tests
+qa/
```

Comment:
`qa/` should no longer be created in current master. If this is still the case, we should rather find the source why this was created then just ignoring it.